### PR TITLE
Merge pull request

### PR DESCRIPTION
**Pull request by sarulathadurai**

### CODE REVIEW

File: content/participants/ttjc@sarulathadurai/index.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+- [Twitter](https://twitter.com/sarulat80587902)
+- [LinkedIn](https://www.linkedin.com/in/sarulatha-annadurai-060226180/)
+
+
+
+### projects
+
+not a lot of projects are done. I will keep updating it with real links when done.
+
+My projects: I don't have any projects as of now but I will update it as and when I complete my projects.
```
Comment (gkanishk) [inline content/participants/ttjc@sarulathadurai/index.md:42]: change this section.

Comment (sarulathadurai) [reply content/participants/ttjc@sarulathadurai/index.md]: changes and conflicts are resolved sir

File: content/participants/ttjc@sarulathadurai/index.md [hunk 2]
```diff
@@ -107,3 +113,33 @@ _hosted link_ https://dev.to/sarulathadurai/how-to-use-map-reduce-and-filter-373
...
+
+### blogs
+
+same as projects, I will keep updating my blogs as and when done.
+
+#### why I liked GraphQL over REST?
+
+<!_description_ I will write something really nice here so that you feel like reading my blog.>
+
+<!_link_ https://dev.to/some-imaginary-link>
```
Comment (gkanishk) [inline content/participants/ttjc@sarulathadurai/index.md]: Please remove this cample part!!

Comment (sarulathadurai) [reply content/participants/ttjc@sarulathadurai/index.md]: I'm sorry sir, didn't noticed that.Now done sir

**Comment by gkanishk** [comment]
Awesome now you can add your finalist file check [faq](https://2020.teamtanay.jobchallenge.dev/finalist-entry/) or #935 for details

**Comment by gkanishk** [comment]
Perfect!

**Pull request merged**